src/Components/OATPropertyEditor/PropertyList.tsx

```diff
@@ -277,15 +277,15 @@ export const PropertyList = ({
                         }}
                         ref={addPropertyLabelRef}
                     >
-                        <ActionButton>
+                        <div>
```

Comment:
I think we wanted to make this a button so it was correct from an accessibility perspective? Do we need to change it to a div?